### PR TITLE
Newsweek

### PR DESCRIPTION
**Pull request by dannkkie**
Please review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,2168 @@
...
+            </div>
+          </div>
+        </div>
+        <!-- EDITOR'S PICK END -->
+
+        <!-- FEATURED SLIDESHOWS START -->
+        <div class="col-12 secDivisor">
+          <div class="divisorImg">
+            <img
+              src="images/doublelittleN.png"
```
Comment (MauricioRobayo) [inline index.html]: Check this image! It is not loading, the path might be wrong, or the image might be missing.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,2168 @@
...
+        <div class="container-fluid col-12 removingPadding">
+          <div class="container-fluid featuredSlides">
+            <div class="col-12 col-md-4 inMag editorsSec">
+              <img
+                src="assets/images/placeholder.jpg"
+                alt="Filler image for the Newsweek Page"
+              />
+              <a href="#" class="numberOfSlides"
+                ><img
+                  src="images/numberImg.JPG"
```
Comment (MauricioRobayo) [inline index.html]: Same for this one!

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
Looking great so far! You have put a lot of hard work into this and it shows.

A few enhancements before continuing:

- [x] Please make the README.md more descriptive, it will make your project stand out even more.
- [x] Delete commented code on style.css, this should be production code.
- [x] Your header is not displaying on the mobile view, please fix it so it shows as the original one.

Webp net-resizeimage (2)

**Status**: Request changes

File: index.html (hunk 1 above)
Comment (dannkkie) [reply index.html]: Changes made and pushed

File: index.html (hunk 2 above)
Comment (dannkkie) [reply index.html]: Changes made and pushed

**Comment by dannkkie** [comment]
Please review again i have made the changes

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
It's almost there 👍 , but there is still an issue with the header, not displaying the logo on mobile view, I am testing this on Chrome, could you check that again please 😄 :

Webp net-resizeimage (3)

**Comment by dannkkie** [comment]
Fixed please review again
small-screen

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
### Hi Edmund!

### Wow very nice job! Readme file is very descriptive now!!! Project looks! :+1:  

### But steel needs little changes before the project can be approved

####  Please check responsiveness of your project

3huui7

####  Also you have a lot css properties but you can use Bootstrap classes instead of it

Screenshot from 2019-11-28 23-00-43

* For example padding and margin. I hope you can find this part of [documentation](https://getbootstrap.com/docs/4.0/utilities/spacing/) useful [Spacing](https://getbootstrap.com/docs/4.0/utilities/spacing/)



### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*

*Happy coding 😄*

File: index.html [hunk 3]
```diff
@@ -0,0 +1,2168 @@
...
+
+              <div class="insideOpinion">
+                <a href="#" class="opinionImg"
+                  ><img src="assets/images/tom-van-laer.png" alt="Tom Van Laer"
+                /></a>
+                <div class="opinionText">
+                  <a href="#" class="opinionTitle">
+                    <h4>Climate Change Is Good</h4>
+                  </a>
+                  <p class="opinionSub">BY TOM VAN LAER/p></p>
```
Comment (Anna-Myzukina) [inline index.html]: Line 756 kindly recommended to delete /p>

File: index.html [hunk 4]
```diff
@@ -0,0 +1,2168 @@
...
+                        src="assets/images/nw-small.png"
+                        alt="Little Sign Up Logo Newsweek"
+                        id="boxImage"
+                      />
+                      <h2 id="boxTitle">SIGN UP FOR OUR NEWSLETTER</h2>
+                    </div>
+
+                    <a href="#" id="boxSignUp">SIGN UP ></a>
+                    <br />
+                    <a href="#" id="boxUpdate">Update your preferences >></a>
```
Comment (Anna-Myzukina) [inline index.html]: Line 965 kindly recommended to delete one of these symbols > after "preferences"

File: index.html [hunk 5]
```diff
@@ -0,0 +1,2168 @@
...
+      <!-- THE BOTTOM HALF OF PAGE BEGINS HERE  -->
+      <div id="bottomHalf" class="container-fluid row">
+        <div class="col-12 col-md-3" id="andrewYang">
+          <img
+            src="assets/images/november-29-2019-cover.jpg"
+            alt="Andrew Yank Cover NewsWeek Page"
+          />
+          <a href="#" class="issue">November 29 ISSUE</a>
+          <div class="underImg">
+            <a href="#">SEE ALL FEATURES ></a>
```
Comment (Anna-Myzukina) [inline index.html]: Line 1059 please delete symbol > after "FEATURES"

File: index.html [hunk 6]
```diff
@@ -0,0 +1,2168 @@
...
+            <div class="col-12">
+              <a href="#" class="lastGenres"><h4>Business</h4></a>
+            </div>
+
+            <div class="col-12 col-md-4 inMag editorsSec">
+              <img
+                src="assets/images/placeholder.jpg"
+                alt="Filler image for the Newsweek Page"
+              />
+              <a href="#" class="insi
```
Comment (Anna-Myzukina) [inline index.html]: Line 2060 please delete symbol > after "SUBSCRIBE"

Comment (Anna-Myzukina) [inline index.html]: Line 2085 please delete symbol > after "SIGN UP"

**Review by Anna-Myzukina — CHANGES_REQUESTED** [review]
Project steel needs changes.

File: index.html (hunk 4 above)
Comment (dannkkie) [reply index.html]: fixed

**Comment by dannkkie** [comment]
All issues fixed

**Review by Forison — COMMENTED** [review]
#### Pull request under review by Addo

**Review by Forison — CHANGES_REQUESTED** [review]
# Hi @dannkkie😇,
you have done a great job👌 so far.
* [x] let us fix the navbar, it does not look good at MD.
Screenshot from 2019-11-28 17-13-45

> you can also use bootstrap ```.img-fluid``` on images to make them responsive🙂.
## Kindly fix this and submit the project for review 
Feel free to contact me on [LinkedIn](https://www.linkedin.com/in/forison-addo-558201159/), [Twitter](https://twitter.com/ADDOBOAKYE1) or slack for further clarification if necessary. I would be happy to assist you.
#### Happy coding
Addo Boakye Forison

**Comment by dannkkie** [comment]
Fixed

**Comment by nikasulo** [comment]
# Really great work implementing the quested changes!
You may merge @dannkkie! See you at the top :rocket:

**Comment by dannkkie** [comment]
Thank you for the reviews i appreciate

**Pull request merged**